spec/lib/nokogiri_builder_spec.rb

```diff
@@ -28,19 +28,19 @@
 
       context 'when the list has multiple p children' do
         let(:html) { '<ol><li><p>testing</p><p>test2</p></li></ol>' }
-        let(:expected_html) { "<ol><li><p>testing test2 </p></li></ol>" }
+        let(:expected_html) { "<ol><li><p>testingtest2</p></li></ol>" }
```

Comment:
Out of interest, why did you have to take the space out of this one?

Reply:
Sorry, thats what the ticket is.... 🤦‍♀️ 

But....isn't this then explaining why the extra spaces were there in the first place? If there are 2 paragraphs within the same list item, we'd want them to be spaced apart?

Reply:
This is a bit of an odd one.  I had to remove the space following my change, but so far neither Clair or I have come across any pages where the formatting in a list has been affected in a detrimental way.  I don't know if this spec was added based on something seen in some advice collection pages, or anticipating that a list could be formatted in this way.  

I waited to open this PR until Claire had been through all the issues related to the extra space had been checked as I didn't want to base this change off of looking at a couple of pages.  Claire informs me that through her manual testing that this does resolve the issue, but I'm happy if we'd be more comfortable to leave this for now. I'd just need to flag it as a known issue and get Ste/Adam to agree for us to leave it